src/HackUnlock/model.js

```diff
@@ -0,0 +1,25 @@
+/* exported HUModelGlobal */
+
+const {GObject} = imports.gi;
+const {HBModelBase} = imports.hackyballs.model;
```

Comment:
It feels to me that the first step here is then to lift the `HBModelBase` class from the hackyballs directory and rename it to be more generic, e.g. `ToyAppModel`.

Reply:
All this piece of code should be removed soon, since it should be replace with @xjuan 's new clippy API. I rather just remove it all together then, than refactoring this more.

Reply:
Sounds good